vcd_cli/vcd.py

```diff
@@ -110,4 +110,6 @@ def help(ctx, tree):
     from vcd_cli import vapp  # NOQA
     from vcd_cli import vdc  # NOQA
     from vcd_cli import vm  # NOQA
+    from vcd_cli import role  # NOQA
```

Comment:
please make sure this passes flake8 validation

Reply:
checked. I don't see any failures.

Reply:
The validations are disabled here because # NOQA, please make sure the imports are in alphabetical order

Reply:
Fixed it in https://github.com/anusuyar/vcd-cli/pull/1/commits/cf39eb20a13421e6933f0fa1b586bfa6e8128051